framework/web/UrlManager.php

```diff
@@ -263,6 +256,25 @@ protected function buildRules($ruleDeclarations)
         return $builtRules;
     }
 
+    private function isCacheAvailable()
```

Comment:
The name doesn't reflect what's actually done inside. I'd use `ensureCache()` that returns either `CacheInterface` instance or `null`.